components/apimgt/org.wso2.carbon.apimgt.rest.api.store.v1/src/main/java/org/wso2/carbon/apimgt/rest/api/store/v1/mappings/ThrottlingPolicyMappingUtil.java

```diff
@@ -36,12 +36,12 @@
     /**
      * Converts a List object of Tiers into a DTO
      *
-     * @param throttlingPolicyList a list of ThrottlingPolicy objects
+     * @param throttlingPolicyList a list of ThrottlingPolicy_ objects
```

Comment:
Isn't this throttlingPolicyList a list of Tier objects

Reply:
doc comment for policyLevel is missing

Reply:
https://github.com/wso2/carbon-apimgt/pull/6797